Fix:

pangox.c:282:15: error: no member named 'find_shaper' in 'struct _PangoFontClass'

https://gitlab.gnome.org/GNOME/pango/-/issues/644
https://gitlab.gnome.org/Archive/pangox-compat/-/issues/1
https://aur.archlinux.org/cgit/aur.git/tree/0002-disable-shaper.patch?h=pangox-compat
--- pangox.c.orig	2012-08-27 18:19:32.000000000 -0500
+++ pangox.c	2023-07-20 20:58:03.000000000 -0500
@@ -279,7 +279,6 @@
 
   font_class->describe = pango_x_font_describe;
   font_class->get_coverage = pango_x_font_get_coverage;
-  font_class->find_shaper = pango_x_font_find_shaper;
   font_class->get_glyph_extents = pango_x_font_get_glyph_extents;
   font_class->get_metrics = pango_x_font_get_metrics;
   font_class->get_font_map = pango_x_font_get_font_map;
